src/main/java/ana_raetcaia/IfElseMain.java

```diff
@@ -0,0 +1,43 @@
+package ana_raetcaia;
+
+import java.util.Scanner;
+
+public class IfElseMain {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a num: ");
+        int num = scanner.nextInt();
+
+      if (num > 0) {
+          System.out.println("Numbr is positive");
+
+            if ((num > 0) && (num % 2) == 0 && (num % 3) == 0) {
```

Comment:
why do you need to verify (num > 0) twice ? 
you already did it in the first parent if

Reply:
No need to do verification (num % n) in ()